calamari-common/calamari_common/types.py

```diff
@@ -303,3 +304,4 @@ class BucketNotEmptyError(Exception):
 # List of allowable things to send as ceph commands to OSDs
 OSD_IMPLEMENTED_COMMANDS = ('scrub', 'deep_scrub', 'repair')
 OSD_FLAGS = ('pause', 'noup', 'nodown', 'noout', 'noin', 'nobackfill', 'norecover', 'noscrub', 'nodeep-scrub')
+CACHE_MOD = ('none', 'writeback', 'forward', 'readonly', 'readforward', 'readproxy')
```

Comment:
@xiexingguo  Thank you for your review.I will fix these after the discussion here.